.github/workflows/build_and_test.yml

```diff
@@ -215,7 +215,7 @@ jobs:
     # Ubuntu 20.04. See also SPARK-33162.
     - name: Install Python packages (Python 3.6)
       run: |
-        python3.6 -m pip install numpy 'pyarrow<3.0.0' pandas scipy xmlrunner
+        python3.6 -m pip install numpy 'pyarrow<3.0.0' pandas scipy xmlrunner plotly>=4.8
```

Comment:
@dongjoon-hyun FYI. Would you mind installing plotly at Python 3.8 in the Docker image we use when you find some time?

Reply:
Oh, I missed this comment last week. I only added plotly to Python 3.9 for now. I will add it to Python 3.8 soon.

Reply:
Oh i should have clarified it.now python 3.9 has to have this since we don't run he pyspark tests with python 3.8 anymore in the master branch, and pandas on Spark is only in the master branch.